client/src/components/InlinePanel/index.js

```diff
@@ -209,6 +209,36 @@ export class InlinePanel extends ExpandingFormset {
     );
   }
 
+  addTabIndex(prefix) {
+    /*
+    add tabindex -1 into newly created form if attr not present and 
+    remove attr from old forms on blur event, if not present previously,
+    */
+    const forms = this.formsElt.children(':not(.deleted)');
+    const inlineChildId = 'inline_child_' + prefix;
+    const formsArray = forms.toArray();
+
+    formsArray.forEach(function(node) {
+      const $node = $(node);
+
+      if ($node.attr('id') === inlineChildId) {
+        if (!$node.attr('tabindex')) {
+          $node.attr('tabindex', -1);
+          $node[0].scrollIntoView({ behavior: 'smooth' });
+          $node.focus();
+          
+          // blur event listener to remove tabindex when it loses focus
+          $node.on('blur', function() {
```

Comment:
```suggestion
          $node.one('blur', function() {
```

Let's ensure this listener does not stick around once run once. 

http://api.jquery.com/one/